### ISSUE TITLE
Failed when opening a file containing a pivot table with calculated fields.

### ISSUE THREAD

**Issue by b0bi79** [ClosedXML/ClosedXML#595]
**Do you want to request a *feature* or report a *bug*?**
- [x] Bug
- [x] Feature

**Version of ClosedXML**
0.89.0

**What is the current behavior?**
If I open a file created with MS Excel that contains a pivot table with a calculated field, then I get a System.ArgumentOutOfRangeException error. The column 'Field1' does not appear in the source range.

**Code to reproduce problem:**
```c#
public void Main()
{
    string fileName = @"pivot.xlsx";
    Stream stream = new FileStream(fileName, FileMode.Open, FileAccess.Read);
    XLWorkbook workbook = new XLWorkbook(stream);
}
```
- [x] I attached a sample spreadsheet. 
[pivot.xlsx](https://github.com/ClosedXML/ClosedXML/files/1486999/pivot.xlsx)

**Comment by igitur** [comment]
Did #603 fix this?

**Comment by b0bi79** [comment]
No. To resolve this issue it is necessary to implement the calculated columns in the Pivot Table.

**Comment by igitur** [comment]
Please give me more information about calculated columns and what's required for them. I could have a look at it.
